File: ce/customerengagement/on-premises/snippets/csharp/CRMV8/dataimport/cs/exportmappingsimportmap.cs
// <snippetexportmappingsimportmap>


 // Retrieve the value for validation
 var exportRequest = new ExportMappingsImportMapRequest
 {
     ImportMapId = _newImportMapId.Value,
     ExportIds = true,
 };

 var exportResponse = (ExportMappingsImportMapResponse)_serviceProxy
     .Execute(exportRequest);

 var mappingXml = exportResponse.MappingsXml;

 Console.WriteLine(String.Concat("\nValidating mapping xml for : ", 
     _newImportMapId.Value, ",\nMappingsXml:\n", mappingXml));

// </snippetexportmappingsimportmap>

File: ce/customerengagement/on-premises/snippets/visualbasic/CRMV8/dataimportvb/vb/exportmappingsimportmap.vb
' <snippetexportmappingsimportmap>


          ' Retrieve the value for validation
          Dim exportRequest = New ExportMappingsImportMapRequest With
                              {
                                  .ImportMapId = _newImportMapId.Value,
                                  .ExportIds = True
                              }

          Dim exportResponse = CType(_serviceProxy.Execute(exportRequest), 
              ExportMappingsImportMapResponse)

	Dim mappingXml = exportResponse.MappingsXml

          Console.WriteLine(String.Concat(vbLf &amp; "Validating mapping xml for : ",
                                          _newImportMapId.Value,
                                          "," &amp; vbLf &amp; "MappingsXml:" &amp; vbLf,
                                          mappingXml))

' </snippetexportmappingsimportmap>